<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabWidget</class>
 <widget class="QWidget" name="tabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tabWidget</string>
  </property>
  <widget class="QGroupBox" name="filterBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>491</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string>Active Filters:</string>
   </property>
   <widget class="QTreeWidget" name="filterTree">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>471</width>
      <height>351</height>
     </rect>
    </property>
    <property name="tabKeyNavigation">
     <bool>true</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="uniformRowHeights">
     <bool>true</bool>
    </property>
    <property name="itemsExpandable">
     <bool>false</bool>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="animated">
     <bool>true</bool>
    </property>
    <attribute name="headerVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="headerDefaultSectionSize">
     <number>85</number>
    </attribute>
    <attribute name="headerHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="headerMinimumSectionSize">
     <number>85</number>
    </attribute>
    <attribute name="headerShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <attribute name="headerStretchLastSection">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Column</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Mode</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Filter</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Strict</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>321</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string>Columns Pending Deletion:</string>
   </property>
   <widget class="QListWidget" name="columnList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>301</width>
      <height>351</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionRectVisible">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
